### PR TITLE
Feat/KFS-1189 multiple small changes

### PR DESCRIPTION
**Pull request by Guttz**
Release Notes
-------------

Checklist
---------
1. [CRUCIAL] Is the change for features that are already live in prod?
   * yes: ok

What
----

Product asked for multiple small changes as part of the ongoing bugbash:

-Flag name from cfu to max-cfu
-Compute Pool Create description adjustment
-Default values for cfu adjusted

References
----------

Test & Review
-------------

golden files updated

Open Questions / Follow-ups
---------------------------

### CODE REVIEW

**Review by swist — APPROVED** [review]

File: internal/flink/command_compute_pool_create.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ func (c *command) computePoolCreate(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	cfu, err := cmd.Flags().GetInt32("cfu")
+	cfu, err := cmd.Flags().GetInt32("max-cfu")
```
Comment (brianstrauch) [inline internal/flink/command_compute_pool_create.go:52]:
```suggestion
	maxCfu, err := cmd.Flags().GetInt32("max-cfu")
```

File: internal/flink/command_compute_pool_create.go [hunk 2]
```diff
@@ -14,21 +14,21 @@ import (
...
-				Code: "confluent flink compute-pool create my-compute-pool --cloud aws --region us-west-2 --cfu 2",
+				Code: "confluent flink compute-pool create my-compute-pool --cloud aws --region us-west-2 --max-cfu 2",
 			},
 		),
 	}
 
 	pcmd.AddCloudFlag(cmd)
 	c.addRegionFlag(cmd)
-	cmd.Flags().Int32("cfu", 1, "Number of Confluent Flink Units (CFU).")
+	cmd.Flags().Int32("max-cfu", 5, "Number of Confluent Flink Units (CFU).")
```
Comment (brianstrauch) [inline internal/flink/command_compute_pool_create.go:31]:
```suggestion
	cmd.Flags().Int32("max-cfu", 5, "Maximum number of Confluent Flink Units (CFU).")
```

Comment (brianstrauch) [reply internal/flink/command_compute_pool_create.go:31]: This is also a breaking change, so let's be aware of that for GA!

File: internal/flink/command_compute_pool_create.go (hunk 1 above)
Comment (Guttz) [reply internal/flink/command_compute_pool_create.go:52]: done

**Review by brianstrauch — APPROVED** [review]

**Pull request merged**